frontend/www/js/omegaup/components/problem/CollectionList.vue

```diff
@@ -33,35 +33,42 @@
         ></omegaup-problem-filter-difficulty>
       </div>
       <div class="col">
-        <omegaup-problem-base-list
-          :problems="problems"
-          :logged-in="loggedIn"
-          :current-tags="selectedTags"
-          :pager-items="pagerItems"
-          :wizard-tags="wizardTags"
-          :language="language"
-          :languges="languages"
-          :keyword="keyword"
-          :modes="modes"
-          :columns="columns"
-          :mode="modes"
-          :column="column"
-          :tags="tagsList"
-          :sort-order="sortOrder"
-          :column-name="columnName"
-          :path="`/problem/collection/${level}/`"
-          @apply-filter="
-            (columnName, sortOrder) =>
-              $emit(
-                'apply-filter',
-                columnName,
-                sortOrder,
-                difficulty,
-                selectedTags,
-              )
-          "
-        >
-        </omegaup-problem-base-list>
+        <div v-if="problems.length == 0" class="card-body">
```

Comment:
```suggestion
        <div v-if="!problems || problems.length == 0" class="card-body">
```

esto es lo que está causando las fallas en las pruebas. igual en el otro archivo.